src/pages/notyet.js

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { Link } from "gatsby";
+
+// styles
+const pageStyles = {
+  display: "flex",
+  height: "100vh",
+  justifyContent: "center",
+  alignItems: "center",
+};
+
+const contentStyle = {
+  textAlign: "center",
+  color: "#fff",
+};
+
+const brandStyles = {
+  fontSize: "4rem",
+  fontWeight: "700",
+};
+
+const messageStyles = {
+  fontSize: "1.8rem",
+};
+
+const linkStyles = {
+  marginTop: "2rem",
+  display: "inline-block",
+  padding: ".8rem 4rem",
+  textAlign: "center",
+  background: "#374151",
+  borderRadius: "5rem",
+  fontSize: "1.4rem",
+  filter: "drop-shadow(0 0 10px #374151)",
+};
```

Comment:
Once the layout is added, this will probably either be changed or not needed and removed.

Reply:
made requested changes.